suites/apis/centralizedAuth.js

```diff
@@ -0,0 +1,655 @@
+    const filename_change = {
+      'file_name': 'test_filename'
+    }
+    const gen3_update_response = indexd.do.updateFile(
+        new_gen3_records.fooBarFile, filename_change, users.mainAcct.accessTokenHeader)
+    const abc_update_response = indexd.do.updateFile(
+        new_abc_records.fooBarFile, filename_change, users.mainAcct.accessTokenHeader)
+
+    // delete
+    const gen3_delete_response = indexd.do.deleteFile(
+        new_gen3_records.deleteMe, users.mainAcct.accessTokenHeader)
+    const abc_delete_response = indexd.do.deleteFile(
+        new_abc_records.deleteMe, users.mainAcct.accessTokenHeader)
+
+    // asserts for create
+    chai.expect(
+      gen3_create_success,
+      'uh oh, user without permission was able to register files to indexd under `/gen3`'
+    ).to.be.false;
+    chai.expect(
+      abc_create_success,
+      'user with permission was NOT able to register files to indexd under `/abc`'
+    ).to.be.true;
+
+    // asserts for read
+    fenceQuestions.assertStatusCode(
+        gen3_read_response, 403,
+        msg='should have gotten unauthorized for reading record under `/gen3`'
+    );
+    fenceQuestions.assertStatusCode(
+        abc_read_response, 200,
+        msg='user with permission could not read record under `/abc`'
+    );
+
+    // asserts for update
+    fenceQuestions.assertStatusCode(
+        gen3_update_response, 403,
+        msg='should have gotten unauthorized for updating record under `/gen3`'
+    );
+    fenceQuestions.assertStatusCode(
+        abc_update_response, 200,
+        msg='user with permission could not update record under `/abc`'
+    );
+    chai.expect(
+        abc_read_response, 'update response does not contain file_name'
+    ).to.have.property('file_name');
+    chai.expect(
+        abc_read_response.file_name,
+        'update record response does not have filename we updated it with'
+    ).to.have.string('test_filename');
+
+    // asserts for delete
+    fenceQuestions.assertStatusCode(
+        gen3_delete_response, 403,
+        msg='should have gotten unauthorized for deleting record under `/gen3`'
+    );
+    fenceQuestions.assertStatusCode(
+        abc_delete_response, 200,
+        msg='user with permission could not delete record under `/abc`'
+    );
+});
+
+/*
+   - Test the same thing as above, but use a client's access_token for that user
+*/
+Scenario('Client (with access) with user token (with access) can CRUD indexd records in namespace, not outside namespace @centralizedAuth',
+  async (fence, indexd, users, files) => {
+    return; // FIXME: skip test for now
+    // NOTE: default CLIENT is abc-admin and gen3-admin
+    const tokenRes = await fence.complete.getUserTokensWithClient(users.mainAcct);
+    const accessToken = tokenRes.body.access_token;
+
+    // create
+    const gen3_create_success = await indexd.do.addFileIndices(
+        Object.values(new_gen3_records), apiUtil.getAccessTokenHeader(users.mainAcct.accessToken));
```

Comment:
```suggestion
        Object.values(new_gen3_records), apiUtil.getAccessTokenHeader(accessToken));
```
this typo is in several tests

Reply:
oh! shoot, good catch